cpp/src/io/utilities/hostdevice_span.hpp

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <cudf/utilities/error.hpp>
```

Comment:
Curious about the need for this include, though.

Reply:
ahh, sorry, was confused.  I thought this was the kernel error header :sweat_smile:

Reply:
can't imagine why, the names are so different :confused: